openedx/features/pakx/lms/overrides/utils.py

```diff
@@ -34,6 +37,72 @@
 ]
 
 
+def get_or_create_course_overview_content(course_key, custom_setting=None):
+    course_overview_content, _ = CourseOverviewContent.objects.get_or_create(
+        course_id=course_key, defaults=custom_setting or {}
+    )
+    return course_overview_content
+
+
+def get_course_card_data(course, org_prefetched=False):
+    """
+    Get course data required for home page course card
+
+    :returns (dict): dict of course card data
+    """
+    pakx_short_logo = '/static/pakx/images/mooc/pX.png'
+    course_custom_setting = get_or_create_course_overview_content(course.id)
+
+    if not org_prefetched:
+        course_org = Organization.objects.filter(short_name__iexact=course.org).first()
+        org_logo_url = course_org and course_org.logo
+        org_name = course.org
+    else:
+        org_name = course.custom_settings.course_set.publisher_org.name
+        org_logo_url = course.custom_settings.course_set.publisher_org.logo
+
+    return {
+        'key': course.id,
+        'org_name': org_name,
+        'effort': course.effort,
+        'image': course.course_image_url,
+        'name': course.display_name_with_default,
+        'org_logo_url': org_logo_url or pakx_short_logo,
+        'short_description': course_custom_setting.card_description,
+        'publisher_logo_url': course_custom_setting.publisher_logo_url,
+        'url': reverse('about_course', kwargs={'course_id': text_type(course.id)}),
+    }
+
+
+def get_featured_course_set():
+    """
+    Get featured course-set data in a list of dict
+    :return (list): return list of dict containing course data
+    """
+    feature_course_set_key = configuration_helpers.get_value('feature_course_set_name')
```

Comment:
Suggestion: `feature_course_set_key` should be named `feature_course_set_name`, current name creates confusion in the following expression
`custom_settings__course_set__name__iexact=feature_course_set_key`